pkg/pillar/cmd/baseosmgr/handleverifier.go

```diff
@@ -49,15 +50,85 @@ func lookupVerifierConfig(ctx *baseOsMgrContext, objType string,
 	return &config
 }
 
+// whether the object contains certs reference
+func containsCerts(safename string, ss *types.StorageConfig) bool {
+	cidx := 0
+	// count the number of cerificates in this object
+	if ss.SignatureKey != "" {
+		cidx++
+	}
+	for _, certURL := range ss.CertificateChain {
+		if certURL != "" {
+			cidx++
+		}
+	}
+	// if no cerificates, return
+	if cidx == 0 {
+		log.Infof("containsCerts() for %s, no certificates configured\n", safename)
+		return false
+	}
+	return true
+}
+
+// check the cert object status
+func checkCertsStatusForObject(ctx *baseOsMgrContext, uuidStr string,
```

Comment:
This should be a utility function - Also used in zedmanager??

Reply:
Done